### DISCUSSION TITLE
Help for migrate Google Font to local

### DISCUSSION THREAD

**Discussion by AlexandreGagner** [fomantic/Fomantic-UI#2720] [General] [answered]
Hi everyone,
I read few issues and discussions about switching from Google to local font, and with last version of Fomantic, the Google font name hava not impact on the build.
I always used the src>site>gloabals>site.variable config file to replace "Lato" by another google font and rebuild.
Except since 2.9, I can't get Google Font work proprely local or distant.
I tried :
"
@fontName          : 'Rubik';
@font          : 'Rubik';
@importGoogleFonts : true;
@importFont: true;
"
in site.variable, but it result with buggy font that blink on load and few error in browser console on loading local font.

Anyone have few tips to use local google font ?

**Comment by lubber-de** [comment, marked as answer]
try
```less
@fontName : 'Rubik';
@importGoogleFonts : true;
@importFonts: false;
```

**Comment by MacLake** [comment]
@lubber-de Could you please explain what exactly are the variables `@importFonts` and `@importGoogleFonts`for? Do they download fonts or do they just add `@font-face ` to the css output? I can’t find any documentation about this.

**Reply by lubber-de** [reply]
`@importFonts` is a switch to  (not) create the custom font-face declarations defined by `@fonts`
`@importGoogleFonts` is a switch to add a CSS import of a font family hosted at google declared via the `@googleFontName` variable

Yes, this deserves proper documentation (it never was since SUI days... even if SUI only offers to import google fonts or not)

Some handling is described in the related PR #2359 though
> Disabling the whole import can be done via @importFont: false or setting @font: {} or setting @fontName: '' just as one likes.

**Reply by lubber-de** [reply]
Probably better understood by looking at the releated codeblock

https://github.com/fomantic/Fomantic-UI/blob/ea336397cedf898e6411de4884e258e5cfb39fa9/src/theme.less#L65-L76
